t/errlog.t

```diff
@@ -1111,3 +1111,123 @@ enter 11
 "
 ]
 --- skip_nginx: 3: <1.11.2
+
+
+
+
+=== TEST 29: ringbuf overflow bug
+--- http_config
+    lua_capture_error_log 4k;
+--- config
+    location /t {
+        access_by_lua_block {
+            local errlog = require "ngx.errlog"
+            local msg = string.rep("*", 10)
+
+            for i = 1, 10 do
```

Comment:
Goal of this test was to show, that messages disappear even if there is enough space for them, that's why I check just number. To show it there should be enough messages to fill the buffer (so it could be smaller amount of bigger messages). Then read a single message and then write a single message.